fv3config/cli.py

```diff
@@ -0,0 +1,26 @@
+import argparse
+
+import yaml
+import fsspec
+
+import fv3config
+
+
+def _parse_args():
+    parser = argparse.ArgumentParser("write_run_directory")
+    parser.add_argument(
+        "config", help="URI to fv3config yaml file. Supports any path used by fsspec."
+    )
+    parser.add_argument(
+        "rundir", help="Desired output directory. Must be a local directory"
+    )
+    return parser.parse_args()
+
+
+def write_run_directory():
+    args = _parse_args()
+
+    with fsspec.open(args.config) as f:
+        config = yaml.safe_load(f)
+
+    fv3config.write_run_directory(config, args.rundir)
```

Comment:
As it is right now, `write_run_directory` does not put the yaml file at `fv3config.yml` in the run directory. It's an outstanding bug to move this yaml-writing into that command and out of the `run_docker`/`run_native` commands. For now you could write the config to that filename here.

Reply:
IMO this seems like a bug that should be addressed by write_run_directory since that fix would apply universally.

Reply:
You're more than welcome to fix the issue by fixing `write_run_directory` instead, I just wanted to also give you the "faster" option of writing the file here, if you'd like to leave the `write_run_directory` to a future PR or to someone else.